### PR TITLE
Added server

### PR DESCRIPTION
**Pull request by NigelMcAfee**
Added the server side, in a backend directory. The base get route has been defined for health checking, but no further routes or middleware yet

### CODE REVIEW

File: backend/package.json
Comment (justinpchen94) [inline backend/package.json:1]: let's actually create this file via the `npm init` command

https://weaintplastic.github.io/web-development-field-guide/Development/Frontend_Development/Setting_up_your_project/Setup_Dependency_Managers/Node_Package_Manager/Initialize_NPM_on_a_new_project.html

File: backend/server.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+
+app.get("/", function (req, res) {
+    res.send("SMILE! :D");
+  });
+
+
+app.listen(port, function () {
+    console.log(`SkillSwap server listening on port ${port}`);
+  });
```
Comment (justinpchen94) [inline backend/server.js]: nit: unindent

File: backend/server.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const app = express();
+const port = 8000;
+
+
+app.get("/", function (req, res) {
+    res.send("SMILE! :D");
+  });
```
Comment (justinpchen94) [inline backend/server.js]: nit: unindent

**Review by justinpchen94 — COMMENTED** [review]
first quick set of comments!

File: backend/package.json
Comment (NigelMcAfee) [reply backend/package.json:1]: Thanks & done!

**Review by justinpchen94 — APPROVED** [review]

**Pull request merged**